core/src/test/java/org/apache/iceberg/avro/TestSchemaConversions.java

```diff
@@ -234,7 +243,10 @@ public void testComplexSchema() {
             3,
             "preferences",
             Types.StructType
-                .of(required(8, "feature1", Types.BooleanType.get()), optional(9, "feature2", Types.BooleanType.get()))),
+                .of(required(
```

Comment:
The formatting changes in this file are awkward. What is the rule that we're applying here?

Reply:
Line length was longer than 120 characters here